scripts/e2e/cmd/runner/mfu_one_namespace_one_ingress_test.go

```diff
@@ -129,6 +129,66 @@ var _ = Describe("MFU", func() {
 			Expect(err).To(BeNil())
 		})
 
+		FIt("[retry access check] should be able to wait for the access to be granted", func() {
+			klog.Info("Initializing role client")
+			roleClient, err := getRoleClient()
+			Expect(err).To(BeNil())
+
+			// remove role assignment
+			// output=$(az role assignment list --assignee $identityPrincipalId --subscription $subscription --all -o json | jq -r ".[].id") | xargs
+			klog.Info("Removing all role assignments")
+			err = removeRoleAssignments(roleClient)
+			Expect(err).To(BeNil())
+
+			klog.Info("Deleting AAD Pod identity pod")
+			err = deleteAADPodIdentityPods(clientset)
+			Expect(err).To(BeNil())
+
+			// delete the AGIC pod. This will create the pod
+			klog.Info("Deleting AGIC pod")
+			err = deleteAGICPod(clientset)
+			Expect(err).To(BeNil())
+
+			// wait for 30 seconds
+			klog.Info("Wait for 60 seconds")
+			time.Sleep(60 * time.Second)
+
```

Comment:
how can we check if AGIC pod is in ready status?

Reply:
let me address this in a separate PR

Reply:
Sounds good, please help track with a task